src/components/Footer.tsx

```diff
@@ -0,0 +1,87 @@
+import { Box, Flex, Text, IconButton, Container } from "@chakra-ui/react";
+import { FaGithub, FaInstagram, FaFacebook } from "react-icons/fa";
+
+const Footer = () => {
+  return (
+    <Box
+      as="footer"
+      py={4}
+      position="absolute"
+      bottom="0"
+      height="82"
+      width="100%"
+      bg="brandGray"
+      color="black">
+      <Container maxW="6xl">
+        <Flex justify="space-between" align="center">
+          <Text pl={4}>&copy; 2023 Olivier Meal Planner</Text>
+
+          <Flex justify="center" alignItems="center">
+            <Box as="a" href="/team" mx={10}>
+              <Text
+                _hover={{
+                  borderBottom: "2px solid green"
+                }}>
```

Comment:
I'm not sure why, but Chakra doesn't seem to be able to pass colors using this format. You need to do this instead:
```
              <Text
                _hover={{
                  borderBottomColor: "green",
                  borderBottomWidth: "2px",
                  borderBottomStyle: "solid"
                }}>
```